codecov.yml

```diff
@@ -8,6 +8,12 @@ coverage:
     project:
       default:
         threshold: 0.5%
+      nodes:
+        target: 80%
+        threshold: 0.5%
+        paths:
+          - packages/nodes-base/**
+        informational: false
```

Comment:
`informational` already [defaults](https://docs.codecov.com/docs/commit-status#informational) to `false`